src/components/Sidebar.tsx

```diff
@@ -198,7 +192,11 @@ const Sidebar: FC<SidebarProps> = ({
     }
   };
 
-  const removeRecipe = (recipeID: number) => {
+  const removeRecipe = (
+    e: React.MouseEvent<HTMLDivElement>,
+    recipeID: number
+  ) => {
+    e.stopPropagation();
```

Comment:
Good call!  I'm guessing you have clickable containers on top of clickable containers?

Reply:
Yup, sure do!